lib/form_data.js

```diff
@@ -159,7 +159,7 @@ FormData.prototype._trackLength = function(header, value, options) {
 };
 
 // TODO: Use request's response mime-type
-FormData.prototype._multiPartHeader = function(field, value, options) {
+FormData.prototype._multiPartHeader = function(field, value, options,encodeFieldNamesWithoutQuotes) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (9)](https://www.codacy.com/app/form-data/form-data/file/1728373643/issues/source?bid=2805153&fileBranchId=3053783#l162)